### PR TITLE
metrics: preserve the name stored in the `.tsv`.

### LINKED ISSUE #449: studio: shortern names are not correctly handled in `post_to_studio`.
Updates sent to the studio API (`eval/loss`) don't match the name in the tracked `.tsv` plots after #444 (`loss`).

Plots unaffected by #444 (`dice_multi`) do work as expected:

Captura de pantalla 2023-02-03 a las 12 53 56

### PR DESCRIPTION
**Pull request by daavoo**
Closes #449

### CODE REVIEW

File: tests/test_report.py [hunk 1]
```diff
@@ -45,17 +45,17 @@ def test_get_renderers(tmp_dir, mocker):
     assert len(scalar_renderers) == 1
     assert scalar_renderers[0].datapoints == [
         {
-            "foo/bar": "0",
+            "bar": "0",
```
Comment (daavoo) [inline tests/test_report.py:48]: This matches datapoints returned by `dvc`

**Comment by dberenbaum** [comment]
Sorry @daavoo! I was running into too many other Studio issues to test this properly with live metrics. Thanks for fixing!

**Comment by dberenbaum** [comment]
@daavoo I need to do a follow up on this one.

**Comment by daavoo** [comment]
> @daavoo I need to do a follow up on this one.

Sorry missed the lack of approval

**Pull request merged**